package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ardupilot_gazebo</name>
+  <version>0.0.0</version>
+  <description>Plugins and models for vehicle simulation in Gazebo Sim with ArduPilot SITL controllers</description>
+  <maintainer email="rhys.mainwaring@me.com">Rhys Mainwaring</maintainer>
+  <license>GPL-3.0</license>
+  <author>Rhys Mainwaring</author>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
```

Comment:
probably need some gazebo build depends here , right ?

Reply:
You're right. I think it should be:

```xml
<depend>gz-cmake3</depend>
<depend>gz-sim7</depend>
```

However these don't appear in the ROS index for Ubuntu 22.04 as the recommended config is to use Gazebo Fortress with ROS 2 Humble (although most of the community seem to be using Garden). There is a similar issue with the `ros_gz` bridge.